src/index.js

```diff
@@ -25,7 +25,6 @@ function extend(base, props) {
 
 function setClass(attributes, value, append) {
 	let cl = getClass(attributes);
-	if (attributes.className) delete attributes.className;
```

Comment:
From my tests, here's what happens (post-patch):

If only className is set, works as intended (class rendered once, correctly).
If only class is set, works as intended (class rendered once, correctly)
If both class and className are set, class is ignored and className is rendered correctly (as the class attribute).